packages/react-components/react-charts-preview/src/components/Legends/Legends.test.tsx

```diff
@@ -213,9 +211,9 @@ describe('Render calling with respective to props', () => {
       overflowText: 'OverFlow Items',
     };
     const component = mount(<Legends {...props} />);
-    component.setProps({ ...props });
-    expect(renderMock).toHaveBeenCalledTimes(2);
-    renderMock.mockRestore();
+    component.setProps({ ...props, hideTooltip: true });
+    const renderedDOM = component.findWhere(node => node.prop('hideTooltip') === true);
+    expect(renderedDOM!.length).toBe(1);
```

Comment:
Have you made this change for similar test case in other charts as well ?

Reply:
No, I'm not yet made these changes in other charts. Should I make those changes now.

Reply:
Yes, you can make those as well as these are similar test cases.